src/forcemove.cpp

```diff
@@ -0,0 +1,172 @@
+#include "forcemove.h"
+#include "random.h"
+namespace Faunus {
+
+namespace Move {
+
+/**
+ * @brief Compute a single dimension contribution to a mean square thermal speed of a particle, i.e., compute 〈v²_x〉.
+ *
+ * This is equivalent to a one third of a total mean square thermal speed of a particle in three dimensions,
+ * i.e., 〈v²〉 = 3 × 〈v²_x〉.
+ *
+ * @param mass particle's mass in dalton
+ * @return a mean square thermal speed in ångström squared per picosecond squared
+ */
+template<typename T=double> static inline T meanSquareSpeedComponent(T mass) {
```

Comment:
Does this need to be templated? T=Double should be sufficient.

Reply:
Un-templated and documentation altered.

A side note for templates: Their definitions have to be available in the translation unit they are used in. Putting a static function template in a cpp file thus makes a perfect sense.